cmd/artifact_upload.go

```diff
@@ -29,7 +29,7 @@ const (
 )
 
 var artifactUploadCmd = &cobra.Command{
-	Use:   "upload [flags] ARTIFACT",
+	Use:   "upload [flags] [ARTIFACT]",
```

Comment:
Why this change? We are not making `ARTIFACT` optional.

Reply:
I'm sorry, it's a mistake